pyanaconda/modules/payloads/payload/live_image/installation.py

```diff
@@ -504,3 +505,56 @@ def run(self):
             log.debug(f"Copying {path} to {destination_path}")
             if os.path.exists(path):
                 shutil.copy2(path, destination_path)
+
+class CopyTransientDConfDataTask(Task):
+    """Task to copy transient dconf data from live user to installed system"""
+
+    def __init__(self, sysroot, paths):
+        """Create a new task."""
+        super().__init__()
+        self._sysroot = sysroot
+        self._paths = paths
+        self._uid = int(os.environ.get('PKEXEC_UID'))
+        self._dconf_dump_file = "/etc/dconf/db/distro.d/10-installer"
+
+    @property
+    def name(self):
+        """Name of the task."""
+        return "Export live user DConf config to installed system"""
+
+    def run(self):
+        """Run the task."""
+        destination_path = os.path.join(self._sysroot, self._dconf_dump_file.lstrip('/'))
+        destination_dir = os.path.dirname(destination_path)
+        os.makedirs(destination_dir, exist_ok=True)
+
+        output_lines = []
+        for path in self._paths:
+            log.debug(f"Exporting DConf settings from uid {self._uid} under {path}")
+            process = startProgram(["dconf", "dump", path], stderr=subprocess.PIPE, env_prune=["USER", "LOGNAME", "HOME"], user=self._uid)
+            stdout, stderr = process.communicate()
+
+            if process.returncode != 0:
+                log.debug(f"dconf dump {path} failed: {stderr.decode('utf-8')}")
+                continue
+
+            lines = stdout.decode('utf-8').split('\n')
```

Comment:
I think this should use `execWithCapture`?

Reply:
it looks like there's a `execWithCaptureAsLiveUser`  but using it would require landing https://github.com/rhinstaller/anaconda/pull/5074 first i think.